policyengine_us/variables/gov/states/ky/tax/income/credits/ky_tuition_tax_credit.py

```diff
@@ -0,0 +1,21 @@
+from policyengine_us.model_api import *
+
+
+class ky_tuition_tax_credit(Variable):
+    value_type = float
+    entity = TaxUnit
+    label = "Kentucky tuition tax credits"  # Form 8863-K
+    unit = USD
+    definition_period = YEAR
+    defined_for = StateCode.KY
+
+    def formula(tax_unit, period, parameters):
+        tentative_tax_credit = add(
```

Comment:
```suggestion
        # We do not currently model the limitation to undergraduate studies at Kentucky institutions.
        tentative_tax_credit = add(
```
Since this would require recalculating both entire credits.